experimental/packages/api-logs/src/ProxyLoggerProvider.ts

```diff
@@ -29,23 +28,19 @@ export class ProxyLoggerProvider implements LoggerProvider {
     options?: LoggerOptions | undefined
   ): Logger {
     return (
-      this.getDelegateLogger(name, version, options) ??
+      this._getDelegateLogger(name, version, options) ??
       new ProxyLogger(this, name, version, options)
     );
   }
 
-  getDelegate(): LoggerProvider {
-    return this._delegate ?? NOOP_LOGGER_PROVIDER;
-  }
-
   /**
    * Set the delegate logger provider
    */
-  setDelegate(delegate: LoggerProvider) {
```

Comment:
Why not making these as private as well?

Reply:
I know that what it means to have them as "conventially private" but not sure if this really useful, if we take this approach, we need to add typedoc @internal annotations to avoid having these methods exposed in docs
https://open-telemetry.github.io/opentelemetry-js/classes/_opentelemetry_api-logs.ProxyLoggerProvider.html

Reply:
I went with what the issue recommended we do but we can discuss alternatives if you like. I added the @internal annotation in the meantime

Reply:
I think adding `@internal` is good enough for this here. The goal is to actually remove this whole class from exports eventually, since our end users are not expected to use it directly. This means that the types won't be available, see https://github.com/open-telemetry/opentelemetry-js/issues/5733.